spark-operator/src/main/java/org/apache/spark/k8s/operator/probe/HealthProbe.java

```diff
@@ -100,9 +100,10 @@ private boolean checkInformersHealth(RuntimeInfo operatorRuntimeInfo) {
           } else {
             if (log.isErrorEnabled()) {
               log.error(
-                  "Controller: {}, Event Source: {}, Informer: {} is not in a healthy state",
+                  "Controller: {}, Event Source: {}, Informer: {} is in {}, not a healthy state",
                   controllerEntry.getKey(),
                   eventSourceEntry.getKey(),
+                  informerEntry.getValue().getStatus(),
```

Comment:
I also saw the error message, but it's a little insufficient because we don't know what is the current status.